src/instances/HeadcountInstance.ts

```diff
@@ -411,6 +411,7 @@ export class HeadcountInstance {
      * Starts a headcount.
      */
     public async startHeadcount(): Promise<void> {
+        console.log(`${this._leaderName} started ${this._dungeon.dungeonName} headcount.`);
```

Comment:
Ideally, use `console.info`. I know ESLint has an option where `console.log` is bad but `console.info` is fine.